src/components/shared/SamplesTable/SamplesTable.js

```diff
@@ -171,7 +171,7 @@ export const SamplesTable = ({
             <PageSizes
               pageSize={samplesQuery.limit}
               totalPages={totalSamples}
-              setPageSize={updatePageSize}
+              onPageSizeChange={updatePageSize}
```

Comment:
The local state `sortBy` was removed from `pages/search`, and the prop was updated to `onPageSizeChange` for `PageSizes`. As a result, the passed prop in `SamplesTable` has been adjusted as well. Thank you!